sdk/schemaregistry/Microsoft.Azure.Data.SchemaRegistry.ApacheAvro/tests/Samples/Sample01_ReadmeSnippets.cs

```diff
@@ -11,6 +11,7 @@
 
 namespace Microsoft.Azure.Data.SchemaRegistry.ApacheAvro.Tests.Samples
 {
+    [Ignore("Only verifying that the sample builds")]
```

Comment:
Hmm, you already have all these Ignores on the individual tests. Any reason for having it in both places?

Reply:
I didn't realize I needed it at the class level for the weekly run to not fail. Until now, all the other runs pass. The weekly run seems to have different mechanics than the other pipelines. This whole PR was because the weekly run was failing. I added this attribute last and didn't think to strip out the other attributes since they are all temporary.